perfdash/www/index.html

```diff
@@ -23,14 +23,23 @@ <h1 class="md-display-2">Performance Dashboard</h1>
         </div>
 	<div style="width: 200px; display: inline-block;">
           <md-input-container>
-            <md-select placeholder="TestName" ng-model="controller.testName"  ng-change="controller.testNameChanged()">
-              <md-option ng-value="testName" ng-repeat="testName in controller.testNames">
-                {{testName}}
+            <md-select placeholder="CategoryName" ng-model="controller.categoryName"  ng-change="controller.categoryNameChanged()">
```

Comment:
Sorry, I guess it wasn't clear earlier - I meant `MetricCategory` for this one (`CategoryName` doesn't say much by itself). So basically, JobName > MetricCategory > MetricName.

Reply:
done.